configure

```diff
@@ -161,23 +160,25 @@ checklib()
     echo_n "library $1: "
     rm -f tmp.c tmp.out
     echo "int main() { return 1; }" >> tmp.c
+    echo "" > tmpml.ml
     r=1
-    $cc $ccopt $ldflags $cclib tmp.c -l$1 -o tmp.out >/dev/null 2>/dev/null || r=0
+    $ocamlc -ccopt "$ccopt $ldflags $cclib" tmp.c tmpml.ml -cclib -l$1 -o tmp.out >/dev/null 2>/dev/null || r=0
```

Comment:
I'm not sure this invocation actually links the compiled C code.  On the other hand, it may not matter because it might be enough to just link the empty ML file with `-custom` and `-cclib -l$1` to force the library to be linked.

Reply:
Ouch! Indeed, I'd not done a "failing" test. Version doing exactly that pushed (and tested with an "incorrect" name for `-lgmp`)

Reply:
Perfect timing!  I was about to run some tests myself...